core/build.rs

```diff
@@ -19,13 +19,19 @@
 // DEALINGS IN THE SOFTWARE.
 
 fn main() {
-    prost_build::compile_protos(
-        &[
+    let cust: protobuf_codegen::Customize = Default::default();
```

Comment:
I think we don't use any of the features described in `lite_runtime`: https://docs.rs/protobuf-codegen/latest/protobuf_codegen/struct.Customize.html#method.lite_runtime

Can you check if we can enable that?

Reply:
Done.